### PR TITLE
docs(main): update basic features alias doc

### PR DESCRIPTION
**Pull request by caohuilin**
### <samp>🤖 Generated by Copilot at 4f64f90</samp>

This pull request improves and synchronizes the documentation for the alias feature in the `@modern-js/main-doc` package. It edits the English and Chinese versions of `alias.mdx` and adds a changeset file to track the patch update.

### <samp>🤖 Generated by Copilot at 4f64f90</samp>

*  Add a changeset file to document the patch update for the `@modern-js/main-doc` package and summarize the changes in both English and Chinese ([link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-1ce5be4b024137cb5d3c2470c511cb17ceac8e49febf688ff3aac57b9b8c1844R1-R7))
*  Improve the wording and clarity of the introduction, the info box, the example code, the instructions, and the reference link for the alias feature in the English documentation ([link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-17d3bd97bf35b2fde4947d6f17c92d178291c31ee5fd96922945292d0baededaL7-R7), [link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-17d3bd97bf35b2fde4947d6f17c92d178291c31ee5fd96922945292d0baededaL17-R21), [link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-17d3bd97bf35b2fde4947d6f17c92d178291c31ee5fd96922945292d0baededaL30-R33), [link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-17d3bd97bf35b2fde4947d6f17c92d178291c31ee5fd96922945292d0baededaL40-R43), [link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-17d3bd97bf35b2fde4947d6f17c92d178291c31ee5fd96922945292d0baededaL53-R55), [link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-17d3bd97bf35b2fde4947d6f17c92d178291c31ee5fd96922945292d0baededaL65-R67))
*  Update the info box, fix the formatting issue, and simplify the instructions for the alias feature in the Chinese documentation to match the English version ([link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-9fbb7a16a8ce4e34a400680a50c7027370cdb9bea84b902c1032e04243446e9bL17-R17), [link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-9fbb7a16a8ce4e34a400680a50c7027370cdb9bea84b902c1032e04243446e9bL30-R30), [link](https://github.com/web-infra-dev/modern.js/pull/3873/files?diff=unified&w=0#diff-9fbb7a16a8ce4e34a400680a50c7027370cdb9bea84b902c1032e04243446e9bL40-R43))

## Checklist

- [x] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/document/main-doc/docs/en/guides/basic-features/alias.mdx [hunk 1]
```diff
@@ -14,11 +14,11 @@ Modern.js allow aliases in JS and CSS, and the following aliases are built in:
 ```
 
 :::info
-When the optional features is enable, the generator will also add built-in aliases dynamically. For example, when BFF is enabled, the `@api` alias will be added by default.
+When enabling optional features, the `new` command will also dynamically add built-in alias. For example, when enabling BFF, the `@api` alias is added by default.
```
Comment (xuchaobei) [inline packages/document/main-doc/docs/en/guides/basic-features/alias.mdx:17]: the `new` command will also dynamically add built-in alias specific to the features.

File: packages/document/main-doc/docs/en/guides/basic-features/alias.mdx [hunk 2]
```diff
@@ -27,17 +27,19 @@ For example, import the modules from the `src/common/` directory in the `src/App
...
-the code in `src/App.tsx`:
+The code in `src/App.tsx` is as follows:
 
 ```ts
 import utils from '@/src/common/utils';
 import '@/src/common/styles/base.css';
 ```
 
-Modern.js also provides a way to config aliases. Adding the `@common` alias as an example. For TypeScript projects, you only need to configure `compilerOptions.paths` under the project root directory `tsconfig.json` as follows:
+Modern.js also provides a way to customize alias. Taking adding the `@common` alias as an example:
```
Comment (xuchaobei) [inline packages/document/main-doc/docs/en/guides/basic-features/alias.mdx:40]: Taking adding the `@common` alias as an example -> For example,  adding the `@common` alias is as follows

This expression is simpler and easy to read.

File: packages/document/main-doc/docs/en/guides/basic-features/alias.mdx [hunk 3]
```diff
@@ -50,7 +52,7 @@ Modern.js also provides a way to config aliases. Adding the `@common` alias as a
 }
 ```
 
-JavaScript project can config by [`source.alias`](/configure/app/source/alias) in `modern.config.js`:
+JavaScript projects can configure [`source.alias`](/configure/app/source/alias) in `modern.config.js`:
```
Comment (xuchaobei) [inline packages/document/main-doc/docs/en/guides/basic-features/alias.mdx:55]: JavaScript projects can configure [`source.alias`](/configure/app/source/alias) in `modern.config.js`:
->

For JavaScript projects,  set [`source.alias`](/configure/app/source/alias) in `modern.config.js`:

File: packages/document/main-doc/docs/en/guides/basic-features/alias.mdx [hunk 4]
```diff
@@ -27,17 +27,19 @@ For example, import the modules from the `src/common/` directory in the `src/App
...
 
 ```ts
 import utils from '@/src/common/utils';
 import '@/src/common/styles/base.css';
 ```
 
-Modern.js also provides a way to config aliases. Adding the `@common` alias as an example. For TypeScript projects, you only need to configure `compilerOptions.paths` under the project root directory `tsconfig.json` as follows:
+Modern.js also provides a way to customize alias. Taking adding the `@common` alias as an example:
+
+For TypeScript projects, just configure `compilerOptions.paths` in the project's `tsconfig.json`:
```
Comment (xuchaobei) [inline packages/document/main-doc/docs/en/guides/basic-features/alias.mdx:42]: configure -> set

**Review by xuchaobei — CHANGES_REQUESTED** [review]

**Review by chenjiahan — APPROVED** [review]

**Review by xuchaobei — APPROVED** [review]

**Pull request merged**